<launch>
    <arg name="output" default="screen" />
    <arg name="memory_limit" default="768" />
    <node name="hfnet_server" pkg="swarm_loop" type="HFNet_server_TF2.py" output="$(arg output)">
        <rosparam>
            nms_radius: 4
            num_keypoints: 200
        </rosparam>
        <param name="model_path" type="string" value="$(find swarm_loop)/config/hfnet_v2" />
        <param name="memory_limit" type="double" value="$(arg memory_limit)" />
    </node>
</launch>
